README.rst

```diff
@@ -416,28 +358,21 @@ following command, assuming you have *git* installed on your ``PATH``::
 
   pip install git+git://github.com/pypiserver/pypiserver.git
 
-Running on Heroku/Dotcloud
---------------------------
-
-https://github.com/dexterous/pypiserver-on-the-cloud contains
-instructions on how to run ``pypiserver`` on one of the supported cloud
-service providers.
```

Comment:
I still argue that the `Running on Heroku/Dotcloud` section and link to this repo should be removed.

- dotcloud is [defunct since 2016](https://www.failory.com/cemetery/dotcloud)
- Heroku is still available but they did have a [bad security incident last year](https://www.theregister.com/2022/05/04/heroku_security_communication_dubbed_complete/) (plus handled it quite badly too) plus they did [shut down free dynos by end of last year](https://help.heroku.com/RSBRUH58/removal-of-heroku-free-product-plans-faq) making it less useful to recommend

When searching [GitHub repos for pypiserver](https://github.com/search?o=desc&q=pypiserver&s=updated&type=Repositories) you can find more recent projects on how to run pypiserver. e.g.:

- how to run pypiserver with docker https://github.com/harryzcy/docker-pypiserver
- a docker compose version https://github.com/joepreludian/pypiserver/
- a rudimentary helm-chart https://github.com/mittwald/pypiserver-helm
- yours for running it on Google AppEngine ;-) https://github.com/deemetree-at-satelligence/pypiserver-appengine

Reply:
Right, true, I do agree with this argument, thanks a lot for the clarification. That's very fair points, so let's indeed remove it, if anyone is asking for it later, just more the reason to provide more up to date guides 👍 😊  Great stuff, thanks for the discussion 👍